tests/unit/parse-test.js

```diff
@@ -9,7 +9,7 @@ describe('Parse', () => {
     describe('literals', () => {
       it('handles language subtags', () => {
         let base = 'https://www.wikidata.org/wiki/Special:EntityData/Q2005.ttl'
-        let mimeType = 'text/turtle'
+        let mimeType = 'text/turtle; charset="UTF-8"'
```

Comment:
I meant that instead of just modifying one line in existing test, which tests something else - *literals: handles language subtags*. Creating a new dedicated test can make more sense.
Imagine that someone introduces change that will break handling content type headers with parameter, but this test results will show failing *literals: handles language subtags*.
I contributed similar fix to handle content type headers with parameter to solid-web-client and wrote 2 dedicated tests https://github.com/solid/solid-web-client/pull/12/files